docs/source/transaction_family_specifications/track_and_trade_transaction_family.rst

```diff
@@ -0,0 +1,747 @@
+
+       // The type-specific value to initialize or update a Property. Only
+       // one of these fields should be used, and it should match the type
+       // specified for this Property in the RecordType.
+       bytes bytes_value = 11;
+       string string_value = 12;
+       sint64 int_value = 13;
+       float float_value = 14;
+       Location location_value = 15;
+   }
+
+   message CreateRecordAction {
+       // The natural key of the Record
+       string record_id = 1;
+
+       // The name of the RecordType this Record belongs to
+       string record_type = 2;
+
+       repeated PropertyValue properties = 3;
+   }
+
+
+A CreateRecord transaction is invalid if one of the following
+conditions occurs:
+
+- The signer is not registered as an Agent.
+- The identifier is the empty string.
+- The identifier belongs to an existing Record.
+- A valid RecordType is not specified.
+- Initial values are not provided for all of the Properties specified
+  as required by the RecordType.
+- Initial values of the wrong type are provided.
+
+
+Finalize Record
+---------------
+
+A FinalizeRecord Transaction sets a Record’s ``final`` flag to true. A
+finalized Record and its Properties cannot be updated. A Record cannot
+be finalized except by its owner, and cannot be finalized if the owner
+and custodian are not the same.
+
+.. code-block:: protobuf
+
+   message FinalizeRecordAction {
+       string record_id = 1;
+   }
+
+
+A FinalizeRecord transaction is invalid if one of the following
+conditions occurs:
+
+- The Record it targets does not exist.
+- The Record it targets is already final.
+- The signer is not both the Record's owner and custodian.
+
+
+Create Record Type
+------------------
+
+The payload of the Transaction that creates RecordTypes is the same as
+the RecordType object itself: it has a name and a list of Properties.
+
+.. code-block:: protobuf
+
+   message CreateRecordTypeAction {
+       string name = 1;
+
+       repeated PropertySchema properties = 2;
+   }
+
+
+A CreateRecordType transaction is invalid if one of the following
+conditions occurs:
+
+- The signer is not registered as an Agent.
+- Its list of Properties is empty.
+- The name of the RecordType is the empty string.
+- A RecordType with its name already exists.
+
+
+Update Properties
+-----------------
+
+An UpdateProperties transaction contains a record_id and a list of
+PropertyValues (see Create Record above). It can only be (validly)
+sent by an Agent authorized to report on the Property.
+
+.. code-block:: protobuf
+
+   message UpdatePropertiesAction {
+       // The natural key of the Record
+       string record_id = 1;
+
+       repeated PropertyValue properties = 2;
+   }
+
+
+An UpdateProperties transaction is invalid if one of the following
+conditions occurs:
+
+- The Record does not exist.
+- The Record is final.
+- Its signer is not authorized to report on that Record.
+- If any of the provided PropertyValues do not match the types
```

Comment:
I'd rewrite the last item to remove the "If".  Here's my attempt, but I'm not sure if it has the same meaning.

     None of the provided PropertyValues match the types specified in the Record's RecordType.

Perhaps it should be "One or more ... do not match ..." instead.   Maybe?